test/functional/signrawtransactions.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2015-2016 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test transaction signing using the signrawtransaction RPC."""
+"""Test transaction signing using the signrawtransactionwithwallet RPC."""
```

Comment:
Test transaction signing using the signrawtransactionwithkey and signrawtransactionwithwallet RPCs.

Reply:
Done.